tests/conformance_dsl/fragment.rs

```diff
@@ -358,7 +358,19 @@ impl WriteAsIon for ProxyElement<'_> {
                     Some(Symbol(symbol)) if symbol.text().is_some() => {
                         let text = symbol.text().unwrap();
                         if text.starts_with("#$:") {
-                            todo!("e-expression transcription")
+                            let macro_id = text.strip_prefix("#$:").unwrap(); // SAFETY: Tested above.
```

Comment:
Personally, I associate `// SAFETY: ...` with uses of `unsafe`.

Reply:
Yea, I can see that; wondering if using `expect` would be a better self-documenting way to highlight that it is a justified unwrap.